app/javascript/packages/form-steps/history-link.tsx

```diff
@@ -0,0 +1,37 @@
+import { useCallback } from 'react';
+import type { MouseEventHandler } from 'react';
+import { Link } from '@18f/identity-components';
+import type { LinkProps } from '@18f/identity-components';
+import useHistoryParam, { getParamURL } from './use-history-param';
+
+type HistoryLinkProps = Partial<LinkProps> & {
+  basePath?: string;
+
+  step: string;
+};
+
+function HistoryLink({ basePath, step, ...linkProps }: HistoryLinkProps) {
```

Comment:
could we get a class comment describing when we're supposed to use this vs `Link` or `Button`?

Reply:
> could we get a class comment describing when we're supposed to use this vs `Link` or `Button`?

Yep, added in 5f3f352 + c39834a.

Essentially, it's our version of something akin to a [React Router](https://reactrouter.com/docs/en/v6/components/link) or [Wouter](https://github.com/molefrog/wouter#link-hrefpath-) link component, to do client-side routing with the form step parameter.

Reply:
Thanks!